index.html

```diff
@@ -35,9 +35,21 @@ <h1 class="pldoc-heading pldoc-element-title hd-4">{{ post.title }}</h1>
             </header>
 
             <div class="pldoc-element-info">
-                <p class="copy copy-base">{{post.info}}</p>
+                <p class="copy copy-base">{{ post.info }}</p>
             </div>
 
+            {% if post.ui-toolkit-component %}
+            <div class="wrapper-alert">
```

Comment:
@andy-armstrong, good question. This view and the ``_layouts/element.html`` have a lot of duplicate markup/content. Its bothered me for a while, but was necessary since the object syntax to populate both Jekyll templates is slightly different - index requires the ``post`` object while element relies on the ``page`` (e.g. ``post.title and ``page.title``).

I'm all for trying to correct this, but would need to do some researching into how other's have overcome it. Is it okay to do as a follow-up piece of work in your opinion?